Gems/CsvSpawner/Code/Source/CsvSpawner/CsvSpawnerCsvParser.cpp

```diff
@@ -66,22 +71,23 @@ namespace CsvSpawner::CsvSpawnerUtils
                 if (isLocalCoordinates)
                 {
                     // handle Z column
-                    if (index_Z < 0)
+                    if (index_Z)
                     {
-                        info.m_transform.SetTranslation(AZ::Vector3(row[index_X].get<float>(), row[index_Y].get<float>(), 0));
+                        info.m_transform.SetTranslation(
+                            AZ::Vector3(row[index_X.value()].get<float>(), row[index_Y.value()].get<float>(), 0));
                     }
                     else
                     {
-                        info.m_transform.SetTranslation(
-                            AZ::Vector3(row[index_X].get<float>(), row[index_Y].get<float>(), row[index_Z].get<float>()));
+                        info.m_transform.SetTranslation(AZ::Vector3(
+                            row[index_X.value()].get<float>(), row[index_Y.value()].get<float>(), row[index_Z.value()].get<float>()));
                     }
                 }
-                else if (isWGSCoordiantes)
+                else if (isWGSCoordinates)
                 {
                     ROS2::WGS::WGS84Coordinate coordinate;
-                    coordinate.m_latitude = row[index_lat].get<double>();
-                    coordinate.m_longitude = row[index_lon].get<double>();
-                    coordinate.m_altitude = row[index_alt].get<double>();
+                    coordinate.m_latitude = row[index_lat.value()].get<double>();
```

Comment:
To you decision I am fine with both.
```suggestion
                    coordinate.m_latitude = row[*index_lat].get<double>();
```

Reply:
I prefer version with ".value()" because asterix sign is more associated with dereferencing pointers/iterators, at least for me.